src/main/java/core/basesyntax/service/ReportServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+
+public class ReportServiceImpl implements ReportService {
```

Comment:
The class implementation should not begin with an empty line. Please remove the redundant empty line at the start of the class definition.